Updates to introductory workflow sections (#349)
darktable-org/dtdocs: content/module-reference/processing-modules/vignetting.md

---

> _**Also note:** This module can produce unnatural-looking results and should be used with care. You may be better to use the [exposure](./exposure.md) module with an [elliptical mask](../../darkroom/masking-and-blending/masks/drawn.md) with a large transition area, perhaps also adding the [color balance rgb](./color-balance-rgb.md) module with the same mask to reduce saturation at the edges._

Reviewer: ```suggestion
_**Also note:** This module can produce unnatural-looking results and should be used with care. Instead, use the [exposure](./exposure.md) module with an [elliptical mask](../../darkroom/masking-and-blending/masks/drawn.md) with a large transition area and, if necessary, use the [color balance rgb](./color-balance-rgb.md) module with the same mask to reduce saturation at the edges._
```
